src/api-service/__app__/onefuzzlib/workers/scalesets.py

```diff
@@ -495,14 +502,17 @@ def sync_scaleset_size(self) -> None:
             return
 
         if size != self.size:
+            # Azure auto-scaled us or nodes were manually added/removed
+            # New node state will be synced in cleanup_nodes
             logging.info(
                 SCALESET_LOG_PREFIX + "unexpected scaleset size, resizing.  "
                 "scaleset_id:%s expected:%d actual:%d",
                 self.scaleset_id,
                 self.size,
                 size,
             )
-            self.set_state(ScalesetState.resize)
```

Comment:
The resize state is no longer necessary since the onefuzz service doesn't handle scaling itself up or down.